dev/ci/ci-basic-overlay.sh

```diff
@@ -79,8 +79,8 @@
 ########################################################################
 # VST
 ########################################################################
-: ${VST_CI_BRANCH:=less_init_plugins}
-: ${VST_CI_GITURL:=https://github.com/letouzey/VST.git}
+: ${VST_CI_BRANCH:=master}
+: ${VST_CI_GITURL:=https://github.com/Zimmi48/VST.git}
```

Comment:
I guess that there is a reason why you put `Zimmi48` instead of `PrincetonUniversity`. Do you think it would make sense to add some kind of explanation (e.g. to indicate when it will be safe (again) to rely on `PrincetonUniversity` rather than `Zimmi48`) ?